src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -429,6 +446,10 @@ bool AsyncRPCOperation_sendmany::main_impl() {
             }
 
             obj = perform_joinsplit(info, outPoints);
+
+            if (jsChange > 0) {
+                changeOutputIndex = find_output(obj, 1);
```

Comment:
[Comment] I would like an assertion here so that with minimal effort a new reader doesn't have to know those "remote invariants", eg: always length 2. (Does not block.)